R/prep_data.R

```diff
@@ -41,12 +41,15 @@
 #'   0.0167. It would be excluded by default or if
 #'   `remove_near_zero_variance` > 0.0166. Larger values will remove more columns
 #'   and this value must lie between 0 and 1.
-#' @param convert_dates Logical or character. If TRUE (default), date columns
-#'   are identifed and used to generate day-of-week, month, and year columns,
-#'   and the original date columns are removed. If FALSE, date columns are
-#'   removed. If a character vector, it is passed to the `features` argument of
-#'   `recipes::step_date`. E.g. if you want only quarter and year back:
-#'   `convert_dates = c("quarter", "year")`.
```

Comment:
This first sentence should read, `Transforms datetime features to "continuous" (default), "categories", or "none".`